source/index.html.md.erb

```diff
@@ -192,6 +192,34 @@ Parameter | Description
 --------- | -----------
 recruitment_cycle | 4-character year (e.g. 2019 for 2019/20 courses)
 
+### Get all changed courses
+
+This endpoint retrieves all courses changed since midnight UTC of the specified day.
```

Comment:
Looks like this needs to be updated to reflect the switch to seconds. I can update it.

Reply:
Fixed